src/components/album/AlbumChat.js

```diff
@@ -1,23 +1,32 @@
-import React, { useEffect } from 'react';
+import React, { useEffect, useState } from 'react';
 import io from 'socket.io-client';
 
 const AlbumChat = ({ apiUrl }) => {
-  useEffect(() => {
+  const [message, setMessage] = useState('');
+
+  // Nandle sending a chat message
+  const handleSendMessage = () => {
     const socket = io(process.env.REACT_APP_SOCKET_BASE_PATH);
```

Comment:
Oh one thing I missed earlier; is that `REACT_APP_SOCKET_BASE_PATH` should be `http://localhost:8000` not `http://localhost:8000/api/v1` (same for whatever equivalent value is in Render.com env)